### ISSUE TITLE
Parameters always the same value

### ISSUE THREAD

**Issue by Toernblom** [Aircoookie/Espalexa#63]
https://gyazo.com/7ae2c2d0eea7841fd0dff87e541299ad

The method is being called everytime i ask alexa to ether turn the thing on or off or change intensity / color. However, the values never change and the brightness is always 0 and the rgb is always 16702800. What is the issue here?

**Comment by Aircoookie** [comment]
Hi!
Probably #6. You can try installing EspAsyncTCP and EspAsyncWebServer and then including `#define ESPALEXA_ASYNC` before `#include <Espalexa.h>`. This problem can occur when the network includes an Echo Dot and the ESP8266 core version is newer than 2.3.0.

**Comment by Toernblom** [comment]
Alright, I'm currently using ESP32 1.0.2. I'll try downgrading to 1.0.0.

**Comment by Aircoookie** [comment]
Ah, sorry, i assumed you were using ESP8266. For ESP32, install AsyncTCP and EspAsyncWebserver and add the include as i described, downgrading doesn't help. I can't fix this issue, the Echo is sending a malformed packet :(

**Comment by Toernblom** [comment]
Oh no worries. Oohhh okey, How can it send a malformed packet though? that's quite odd.. Well thank you for your contribution anyway, it's still amazing!

**Comment by Aircoookie** [comment]
Well, ask Amazon. It's sending JSON with the wrong content type `x-www-urlencoded`, so the ESP libraries (rightfully) have difficulties parsing it. Hope you can get it to work!

**Comment by Toernblom** [comment]
Oh okey..

Well damn, for anyone reading this here is the fix ELI5: 
Download these 2: 
https://github.com/me-no-dev/ESPAsyncWebServer
https://github.com/me-no-dev/ESPAsyncWebServer
and put them in your arduino library then add:

#define ESPALEXA_ASYNC
before include <Espalexa.h>

It was an really easy fix so thank you for this :)

**Comment by m4rvin42** [comment]
Thanks a lot for this solution, I had the exact same issue.
You spared me probably hours of searching in the wrong place. I would never have guessed, that the Echo is sending wrong data.
